composer.json

```diff
@@ -1,6 +1,6 @@
 {
 	"require-dev": {
-		"nextcloud/ocp": "dev-stable25@dev",
+		"nextcloud/ocp": "dev-master",
```

Comment:
Can you revert this specific change, currently we still cover multiple nextcloud versions and want to run the psalm checks against the ocp package towards the lowest supported.

Reply:
Class `OCP\Share\Events\BeforeShareCreatedEvent` does not exist on `"nextcloud/ocp": "dev-stable25@dev",` . We are listening to this event [here](https://github.com/nextcloud/notes/pull/1146/files#diff-4eb824426a8cffb8b33b62b57744c69f32ea30ea755249a7a6170d9402885debR31)

Reply:
Right, it is only available since 28. 

Let me check if there is a fallback we can use for that then

Reply:
I pushed a fallback and added the event to the stub for the static analysis tests.

@luka-nextcloud I just noticed that my previously added commit was overwritten with yours and repushed that. Quick hint, if you use `git push --force-with-lease` instead of just a force push you will get notified about changes that have been made in the meantime. https://git-scm.com/docs/git-push#Documentation/git-push.txt---no-force-with-lease

Reply:
Ok, thanks for your tip.